docs/pages/jupedsim/jupedsim_requirements.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Requirements
+subtitle: Dependencies
+summary: In order to compile JuPedSim it is necessary to first install the required libraries.
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jupedsim_requirements.html
+las_updated: Dec 20, 2019
+---
+
+
+## Supported compilers
+
+Any compiler with support for C++11.
+
+Tested with
+
+- `g++` (linux/osx)
```

Comment:
We test the following Compiler at the moment: 
* g++-8 on Linux
* clang++-8 on Linux
* appleclang++-11 on MacOS
* MSVC 19 on Windows